article/sections/_1_intro_generic.tex

```diff
@@ -0,0 +1,48 @@
+\section{Introduction}
+\label{sec:intro}
+
+In transportation, we often build models that are used for policy evaluation.
+Specifically, we often use behavioral models to evaluate the impact of external interventions on travel outcomes. 
+These evaluations are excplicitly causal: we are interested in how the 
+system reacts to \textit{interventions}. 
+Yet, when these models are 
+developed, causality is often not considered. Furthermore, when 
+causal concepts are accounted for, the process is done implicitly without a 
+formal framework. 
+
+This chapter is concerned with filling this gap. 
+Specifically, we focus on addressing the existing disconnect between the fields of travel demand modeling and causal inference. 
+The chapter is motivated by the current lack of use of methods and findings from 
+the causal inference literature in travel demand modeling. 
+
+While the field of transportation demand modeling could benefit greatly from 
+incorporating causal inference techniques, there are barriers that have slowed this integration. 
+These barriers stem from the difference between the types of problems transportation demand modelers deal with and those that are typically studied in the causal inference literature. 
+Perhaps the fundemental difference is that demand modelers are typically trying to forecast the impacts of policies that haven't been implemented or seen before.
```

Comment:
I would add a contrast example from the side of causal inference.
I also would not say that causal inference doesn't try to do that. Maybe observational causal inference does but not all causal inference.

```suggestion
An example of this difference is that travel demand modelers are typically trying to forecast the impacts of policies that have not been implemented yet.
```

Reply:
agreed, the people from structural economics would probably fall over in horror if they felt disregarded (imagine Nobel Laureate Jim Heckman who explicitly writes about the need to forecast policy impacts for unseen policies).